README.md

```diff
@@ -90,14 +90,14 @@ This will kick off a build [in CircleCI](https://circleci.com/gh/mozilla/fxa-ema
      this instance will handle requests from.
      Valid values are `sendgrid` and `socketlabs`.
 
-   * `SQS_SUFFIX`:
-     The suffix for queue names,
-     e.g. `dev` or `stage`.
-     This setting assumes that our queues
-     are always named like
-     `fxa-email-bounce-${SQS_SUFFIX}`.
-     If that's not the case,
-     we'll need to change the code.
+   * `BOUNCE_QUEUE_URL`:
+     Name of the bounce SQS queue
```

Comment:
`Name of` sounds potentially misleading to me here. We really mean the full URL like `https://sqs.us-east-1.amazonaws.com/.../fxa-email-bounce-dev` or whatever, don't we?